### ISSUE TITLE
[fvt]2.12.1 "Smartmatch is experimental" warning in perl5.8.2 while using xcatprobe

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1377]
env:
sles12
perl version:
c910f04x30v01:/opt/xcat/bin # perl -v

This is perl 5, version 18, subversion 2 (v5.18.2) built for x86_64-linux-thread-multi

Copyright 1987-2013, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.

build:c910f04x30v01:/opt/xcat/bin # lsdef -v
lsdef - Version 2.12.1 (git commit fddd78e1ec6a9d59bc921bef24e300e8d94f69e4, built Sun Jun 19 06:45:32 EDT 2016)
How to reproduce:
```c910f04x30v01:/opt/xcat/bin # vi xcatprobe 
c910f04x30v01:/opt/xcat/bin # ./xcatprobe -l
Smartmatch is experimental at ./xcatprobe line 63.
Smartmatch is experimental at ./xcatprobe line 172.
Supported sub commands are:
discovery         Discovery Check. 
switch-macmap     To retrieve MAC address mapping for the specified switch, or all the switches defined in switches table 
                  in xCAT db. 
xcatmn            After xcat installation, use this command to check if xcat has been installed correctly and is ready 
                  for use. Before using this command, please install tftp, nslookup and wget commands ahead. The platform 
                  supported are redhat, sles and ubuntu. 
detect_dhcpd      detect_dhcpd can be used to detect the dhcp server in a network for a specific mac address. Before using 
                  this command, please install tcpdump command ahead. The operating system supported are redhat, sles, 
                  ubuntu and debian. 


```

**Comment by junxiawang** [comment]
@hu-weihua 
please ref :+1: https://github.com/mludvig/smtp-cli/issues/3

**Comment by daniceexi** [comment]
@hu-weihua I am curious that on which OS there's no this warning? (I mean from which perl version)
I guess we can add following sentence to press the warning before fixing it.
~~~~
no if $] >= 5.018, "experimental::smartwatch";
~~~~

**Comment by hu-weihua** [comment]
This issue is fixed by pull request #1472 , @junxiawang  , could you please verify it? thanks

**Comment by junxiawang** [comment]
verified .
